Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -1434,10 +1435,17 @@ static Vector<String> authoredGridTrackSizes(Node* node, Style::GridTrackSizingD
         }
     }
 
+    if (cssValue && cssValue->hasVariableReferences()) {
+        Style::Extractor extractor(element);
+        auto computedValue = extractor.propertyValue(directionCSSPropertyID);
```

Comment:
nice! this is much cleaner 😃

thinking about this a bit more, i think it might actually be nice to show that the value is originally `auto` so that developers have some visual clue as to what things may change if the container adjusts

perhaps something like `auto - 100px` or _`(100px)`_ (either italicized and/or in a lightgrey color to indicate it's not a "normal" value)?

since this PR is already an improvement as-is i'm ok doing this as a followup if you'd rather keep things cleaner (and if so please just file a bug with a FIXME comment here)